x-pack/test/security_solution_cypress/cypress/e2e/investigations/timelines/data_providers.cy.ts

```diff
@@ -28,16 +28,12 @@ import {
 } from '../../../tasks/timeline';
 import { getTimeline } from '../../../objects/timeline';
 import { hostsUrl } from '../../../urls/navigation';
-import { scrollToBottom } from '../../../tasks/common';
 
-// Failing in serverless
-// FLAKY: https://github.com/elastic/kibana/issues/169396
-describe.skip('timeline data providers', { tags: ['@ess', '@serverless'] }, () => {
+describe('Timeline data providers', { tags: ['@ess', '@serverless'] }, () => {
   beforeEach(() => {
     login();
     visitWithTimeRange(hostsUrl('allHosts'));
     waitForAllHostsToBeLoaded();
-    scrollToBottom();
     createNewTimeline();
```

Comment:
we should remove the `should('be.visible')` [here](https://github.com/michaelolo24/kibana/blob/unskip-discover-state-tests/x-pack/test/security_solution_cypress/cypress/tasks/timeline.ts#L305) and [here](https://github.com/michaelolo24/kibana/blob/unskip-discover-state-tests/x-pack/test/security_solution_cypress/cypress/tasks/timeline.ts#L306) as well as the `force: true`

Reply:
Did in the other PR here: https://github.com/elastic/kibana/pull/173308